casr/src/bin/casr-java.rs

```diff
@@ -56,6 +56,17 @@ fn main() -> Result<()> {
                 .value_name("FILE")
                 .help("File with regular expressions for functions and file paths that should be ignored"),
         )
+        .arg(
+            Arg::new("sub-tool")
+                .long("sub-tool")
+                .default_value("casr-san")
+                .action(ArgAction::Set)
+                .value_parser(clap::value_parser!(PathBuf))
```

Comment:
validate option, search in PATH

Reply:
You can also parse the absolute path here

Reply:
Done